src/components/range-inputs.tsx

```diff
@@ -196,20 +233,18 @@ const RangeInputHOC = ({ inputType, classes }: RangeInputHOCProps) => {
     }
 
     /**checks if both inputs are empty and disables the submit button */
-    const handleChange = (value: string) => {
+    const handleChange = () => {
         const formatedValues = type === 'timestamp' ? formatTimeValues()
             : { fromVal: fromRef?.current?.value || '', toVal: toRef?.current?.value || '' };
 
         console.log(formatedValues);
 
-        if (!formatedValues.fromVal && !formatedValues.toVal) disableSubmitBtn();
-        else enableSubmitBtn();
+        toggleSubmitBtn(!formatedValues.fromVal && !formatedValues.toVal);
```

Comment:
When one of the values is invalid, we want the submit button to be disabled. I think the condition needs to be updated to check `!validatedResult` also which mean this will have to be called after calling `validateValue` for both values.

Reply:
```
const validatedResult = validateValue(formatedValues.fromVal) && validateValue(formatedValues.toVal);

const areBothFieldsEmpty = !formatedValues.fromVal && !formatedValues.toVal;

toggleSubmitBtn(areBothFieldsEmpty || !validatedResult);
```